package.json

```diff
@@ -157,7 +162,7 @@
     "rollup-plugin-angular-aot": "^0.0.4",
     "run-sequence": "^1.2.2",
     "rxjs": "5.0.0-beta.12",
-    "sass-loader": "4.0.2",
+    "sass-loader": "6.0.2",
```

Comment:
I've run into several problems with sass-loader 6.x, I have had to down grade to 4.1.1. Is there a reason to use 6.x?